client/src/components/Common/EmptyView/index.js

```diff
@@ -11,27 +11,29 @@ import './styles.css';
 const EmptyView = ({ section, children, suggestion }) => {
   const defaultDescription = (
     <React.Fragment>
+      <p>This section is dedicated to the information about my {section}.</p>
       <p>
-        This section is dedicated to the information about my {section}.
+        For the moment you have not discovered anything regarding this topic. To
+        get some info - ask the bot right question. When the bot gives you some
+        related answer - the information will appear on this page. You will have
+        a progress bar that will help you to understand how much info you have
+        opened already.
       </p>
       <p>
-        For the moment you have not discovered anything regarding
-        this topic. To get some info - ask the bot right question.
-        When the bot gives you some related answer - the information will
-        appear on this page. You will have a progress bar that will help you to
-        understand how much info you have opened already.
-      </p>
-      <p>
-        For example, <b>{'"'}{suggestion}{'"'}</b> sounds like a great question to start,
-        but it can be something else too, just use your imagination ;)
+        For example,{' '}
+        <b>
+          {'"'}
```

Comment:
I do not like this workaround, probably is better to use &quot;, &ldquo;, &#34; or &rdquo; here 🤔

Reply:
FIXED, together with the {' '}